yarn-project/simulator/src/public/transitional_adapters.ts

```diff
@@ -146,16 +145,6 @@ export function generateAvmCircuitPublicInputs(
     }
   }
 
-  const nullifiersFromPrivate = revertCode.isOK()
-    ? mergeAccumulatedData(
-        avmCircuitPublicInputs.previousNonRevertibleAccumulatedData.nullifiers,
-        avmCircuitPublicInputs.previousRevertibleAccumulatedData.nullifiers,
-      )
-    : avmCircuitPublicInputs.previousNonRevertibleAccumulatedData.nullifiers;
-  avmCircuitPublicInputs.accumulatedData.nullifiers = assertLength(
-    mergeAccumulatedData(nullifiersFromPrivate, avmCircuitPublicInputs.accumulatedData.nullifiers),
-    MAX_NULLIFIERS_PER_TX,
-  );
   const msgsFromPrivate = revertCode.isOK()
```

Comment:
Don't need to patch public inputs with nullifiers from private because we are tracing them alongside everything else.